content/php-client/getting-started.md

```diff
@@ -0,0 +1,68 @@
+# Getting started
+
+The PHP client allows to use easily the API in your PHP project. It uses PHP arrays in input and output.
+
+Want to be ready quickly? Read the following instructions!
+
+## Requirements
+
+* PHP >= 5.6
+* Composer
+
+## Installation
+
+`api-php-client` uses [Composer](http://getcomposer.org).
+The first step to use `api-php-client` is to download composer:
+
+```bash
+$ curl -s http://getcomposer.org/installer | php
+```
+We use HTTPPlug as the HTTP client abstraction layer. If you want to know more about this, it's documented [here](/php-client/http-client.html).
+In this example, we will use [Guzzle](https://github.com/guzzle/guzzle) v6 as the HTTP client implementation.
```

Comment:
Actualy, it's pretty technical. It means that you can use Guzzle v6 in your project, but if you prefer, you can use directly a Curl client or Guzzle v5. There are a lot of adapters.

Most of the time, it's not a big deal. But by forcing to use a specific HTTP client, such as Guzzle6, you can have conflict in your depencies because your project is already Guzzle v5.

With our client, it's not a problem.